Library/Formula/dovecot.rb

```diff
@@ -23,6 +24,11 @@ class Dovecot < Formula
     sha256 "82892f876d26008a076973dfddf1cffaf5a0451825fd44e06287e94b89078649"
   end
 
+  resource "stemmer" do
+    url "https://github.com/snowballstem/snowball.git",
+      :revision => "9b58e92c965cd7e3208247ace3cc00d173397f3c"
+  end
```

Comment:
Sorry, I don't know what that means. I merely copied this stanza from another formula in the library.